evm-contracts/src/v0.6/dev/SimpleReadAccessController.sol

```diff
@@ -4,10 +4,10 @@ import "../Owned.sol";
 import "./AccessControllerInterface.sol";
 
 /**
- * @title SimpleAccessControl
+ * @title SimpleReadAccessController
  * @notice Allows the owner to set access for addresses
```

Comment:
I suggest adding a comment clarifying the behavior, e.g.

> SimpleReadAccessController gives access to:
> - any externally owned account (note that offchain actors can always read any contract storage regardless of onchain access control measures, so this does not weaken the access control while improving usability)
> - accounts explicitly added to an access list
>
> SimpleReadAccessController *is not* suitable for access controlling writes since it grants any externally owned account access!